code/CLI/CLIEditor.py

```diff
@@ -179,54 +181,400 @@ def getCommand(model:UMLModel, command:str, arguments:list = []) -> Command:
 
 ##########################################################################
 
-def REPL():
+class REPL(cmd.Cmd):
     """Read Eval Print Loop for the UMLEditor program
 
     A constantly running loop for the user to input commands to modify 
     the state of a UMLModel object. 
     """
+    intro = "This is the command-line interface for UMLEditor. \nPush 'Tab' to: view commands / help with completion."
+    prompt = "UMLEditor> "
```

Comment:
Should we add the prompt color back to the prompt?

Reply:
We certainly can. I don't know how though since intro is a member of the cmd class

Reply:
you can add the color codes to the prompt string and when it prints out the prompt, then it will print the codes too.